setup.py

```diff
@@ -42,6 +46,9 @@ def read(*filenames, **kwargs):
         'License :: OSI Approved :: MIT License',
         'Programming Language :: Python',
         'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
```

Comment:
Ah, k. Thanks for the explanation =)

I think everything is fine as is but wanted to make sure :+1: